Loosen mastodon.py requirement to prevent delay of mastodon 3.1 support.

Index: requirements.txt
--- requirements.txt.orig
+++ requirements.txt
@@ -1,5 +1,5 @@
 click>=6.7
-Mastodon.py==1.5.0
+Mastodon.py>=1.5.0
 colored>=1.3.93
 humanize>=0.5.1
 emoji>=0.4.5
